src/screens/AdjustStock/index.tsx

```diff
@@ -6,75 +6,75 @@ import InputBox from '../../components/InputBox';
 import Button from "../../components/Button";
 import showPopup from "../../components/Popup";
 import {useNavigation, useRoute} from "@react-navigation/native";
-import SelectDropdown from "react-native-select-dropdown";
 import RenderData from "../../components/RenderData";
 import {RootState} from "../../redux/reducers";
 import {stockAdjustments} from "../../redux/actions/products";
+import DropDownPicker from 'react-native-dropdown-picker';
 
 const reasonCodes = [
```

Comment:
These look like `listInventoryAdjustmentReasonCodes` from `ReasonCodes` from core. Could we pull it from api?

Reply:
@awalkowiak I'll add a separate ticket for that.